spec/lib/oauth/token_spec.rb

```diff
@@ -91,6 +91,16 @@ module OAuth
           expect(AccessToken).to receive(:by_token).with('token')
           Token.authenticate double, token
         end
+
+        context 'revoke old token on first access token use' do
+          it 'calls revoke previous refresh token if token was found' do
+            Doorkeeper.configuration.stub(:refresh_token_revoked_on_use).and_return(true)
+            token = ->(r) { 'token' }
```

Comment:
Unused block argument - `r`. If it's necessary, use `_` or `_r` as an argument name to indicate that it won't be used. Also consider using a proc without arguments instead of a lambda if you want it to accept any arguments but don't care about them.